tests/integration/api_packages_arch_test.go

```diff
@@ -336,6 +336,30 @@ HMhNSS1IzUsBcpJAPFAwwUXSM0u4BjoaR8EoGAWjgGQAAILFeyQADAAA
 			MakeRequest(t, req, http.StatusNoContent)
 		}
 	})
+	t.Run("Package Arch Test", func(t *testing.T) {
```

Comment:
The test name seems strange (indeed the while file is for "package arch test"), will improve it.